apps/balancer-tools/src/app/apr/(utils)/calculatePoolStats.ts

```diff
@@ -129,9 +140,37 @@ function calculateRoundAPR(
       : -1;
 
   return {
-    total: vebalAPR,
+    total: vebalAPR + feeAPR,
     breakdown: {
       veBAL: vebalAPR,
+      fee: feeAPR,
     },
   };
 }
+
+const getFeeApr = async (
+  poolId: string,
+  network: string,
+  from: number,
+  to: number,
+): Promise<number> => {
+  const res = await pools.gql(network).poolSnapshotInRange({
+    poolId,
+    from,
+    to,
+  });
+  const previousRoundEndDate = from + 60;
+  const RoundEndDate = to - 60;
+
+  const previousRoundData = res.poolSnapshots.find(
+    (snapshot) => snapshot.timestamp === previousRoundEndDate,
+  );
+
+  const RoundData = res.poolSnapshots.find(
+    (snapshot) => snapshot.timestamp === RoundEndDate,
+  );
+
+  const feeDiff = RoundData?.swapFees - previousRoundData?.swapFees;
+  const feeApr = feeDiff / RoundData?.pool.totalLiquidity;
```

Comment:
this is the fee "time_periodly" percentage return, right? needs to be annualized for it to make sense. Since `from` and `to` can be variable here, the formula that you probably want is